dev/io.openliberty.jaxws.security_fat.1/publish/files/JaxWsTransportSecurityServer/serverConfigs/withAliasClientCertConfiguration.xml

```diff
@@ -0,0 +1,66 @@
+<server>
```

Comment:
Can you do the same with the server configuration files? Verify that only the ones used by the tests in this project are in this project, so we don't have unnecessary duplicates?